Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM mambaorg/micromamba
```

Comment:
To make ncbi-tools work in a Docker image without running any additional configurations you could add:
```
RUN apt-get update && apt-get install uuid-runtime
```
and then
```
RUN mkdir $HOME/.ncbi
RUN printf '/LIBS/GUID = "%s"\n' `uuidgen` > $HOME/.ncbi/user-settings.mkfg
```
This will trick it into thinking the tools were already configured. Of course, if a user wants to configure some additional things like proxy, they'll need to run vdb-config themselves.

Reply:
Fantastic! Thanks for the suggestion Michal.

Reply:
@misialq to confirm, is this in the place of `vdb-config -i` or a necessary addition to it?

Reply:
Yup @lina-kim, that should _replace_ it :) But again, only in the case where the user doesn't need any actual configuration. If they need to configure anything else they'd need to run the command you mentioned.

Reply:
Amazing! That really clears up my issues, thanks Michal! 🎉